static/app/views/performance/traceDetails/traceViewDetailPanel.tsx

```diff
@@ -0,0 +1,504 @@
+import Link from 'sentry/components/links/link';
+import LoadingIndicator from 'sentry/components/loadingIndicator';
+import {
+  ErrorDot,
+  ErrorLevel,
+  ErrorMessageContent,
+  ErrorMessageTitle,
+  ErrorTitle,
+} from 'sentry/components/performance/waterfall/rowDetails';
+import QuestionTooltip from 'sentry/components/questionTooltip';
+import {generateIssueEventTarget} from 'sentry/components/quickTrace/utils';
+import {Tooltip} from 'sentry/components/tooltip';
+import {PAGE_URL_PARAM} from 'sentry/constants/pageFilters';
+import {IconChevron, IconOpen} from 'sentry/icons';
+import {t, tn} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {EntryBreadcrumbs, EntryType, EventTransaction, Organization} from 'sentry/types';
+import {objectIsEmpty} from 'sentry/utils';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import getDynamicText from 'sentry/utils/getDynamicText';
+import {PageErrorProvider} from 'sentry/utils/performance/contexts/pageError';
+import {generateProfileFlamechartRoute} from 'sentry/utils/profiling/routes';
+import {useLocation} from 'sentry/utils/useLocation';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
+import {isCustomMeasurement} from 'sentry/views/dashboards/utils';
+import DetailPanel from 'sentry/views/starfish/components/detailPanel';
+
+import {transactionSummaryRouteWithQuery} from '../transactionSummary/utils';
+
+import {EventDetail} from './newTraceDetailsContent';
+import {Row, Tags} from './styles';
+
+type DetailPanelProps = {
+  detail: EventDetail | undefined;
+  onClose: () => void;
+};
+
+type EventDetailProps = {
+  detail: EventDetail;
+  location: Location;
+  organization: Organization;
+};
+
+function OpsBreakdown({event}: {event: EventTransaction}) {
+  const [showingAll, setShowingAll] = useState(false);
+  const breakdown = event && generateStats(event, {type: 'no_filter'});
+
+  if (!breakdown) {
+    return null;
+  }
+
+  const renderText = showingAll ? t('Show less') : t('Show more') + '...';
+  return (
+    breakdown && (
+      <Row
+        title={
+          <FlexBox style={{gap: '5px'}}>
+            Ops Breakdown
+            <QuestionTooltip
+              title={t('Applicable to the children of this event only')}
+              size="xs"
+            />
+          </FlexBox>
+        }
+      >
+        <div style={{display: 'flex', flexDirection: 'column', gap: space(0.25)}}>
+          {breakdown.slice(0, showingAll ? breakdown.length : 5).map((currOp, index) => {
+            const {name, percentage, totalInterval} = currOp;
+
+            const operationName = typeof name === 'string' ? name : t('Other');
+            const durLabel = Math.round(totalInterval * 1000 * 100) / 100;
+            const pctLabel = isFinite(percentage) ? Math.round(percentage * 100) : '∞';
+
+            return (
+              <div key={index}>
```

Comment:
We shouldn't really use indexes as keys, I'm surprised we don't have a linting rule for this anymore? Is it possible to use the operation name here or does it result in duplicates? I can't remember off the top of my head if the scope will conflict across rows

https://sentry.io/answers/unique-key-prop/

Makes me think that if we nest it in the `div` it should be fine to have the keys repeat. It's only an issue if this map can produce the same span op

Reply:
Operation names are unique, will use it instead.